src/fhir/resource_helpers.ts

```diff
@@ -62,7 +62,7 @@ export function createPatientObservation(
 export const observationComponent = (coding: Coding, value: Coding | string | boolean): ObservationComponent => {
   const component = new ObservationComponent();
   component.code = { coding: [coding] };
-  if (value instanceof Coding) {
+  if (typeof value === "object") {
```

Comment:
it turns out because I was using a class and these weren't created by a class then it doesn't work. If there's a better way to ensure that an object matches a class signature then I'm all ears!

Reply:
I think I've resolved this in a nice way, just by reordering and then using the typing so that it's clear that the only other option left conforms to Coding